xmtp_mls/src/identity/mod.rs

```diff
@@ -1,2 +0,0 @@
-pub mod v3;
-pub mod xmtp_id;
```

Comment:
remove multiple layers of this identity mod including legacy Identity and use a single file mod instead.